src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -11,16 +11,21 @@
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(UseContractReferences)' == 'true'" >
```

Comment:
If we want UseContractReferences to be true by default, in the `<PropertyGroup>` above we probably want to add,
```
<UseContractReferences Condition="'$(UseContractReferences)' == ''">true</UseContractReferences>
```

And we may also want to change the line below from
```
<ItemGroup Condition="'$(UseContractReferences)' == ''" >
```
to
```
<ItemGroup Condition="'$(UseContractReferences)' != 'true'" >
```